PocketKit/Sources/PocketKit/MyList/SavesContainerViewController.swift

```diff
@@ -1,8 +1,10 @@
 import UIKit
+import SwiftUI
 
 struct SelectionItem {
     let title: String
     let image: UIImage
+    let itemIndex: Int
```

Comment:
just a thought, not a blocker. Might be nice to use an enum type that conforms to Int, so we can know that case saves = 0, case archive = 1. what do you think?

Reply:
okee dokee. I'm not too concerned with int since its only 2 types of screens now, but I'll update it to use an enum

Reply:
ok no prob, just a thought. If it makes things complicated no worries

Reply:
updated and done already. no worries